ui/slider.js

```diff
@@ -440,7 +440,7 @@ return $.widget( "ui.slider", $.ui.mouse, {
 				this._removeClass( "ui-slider-horizontal ui-slider-vertical" )
 					._addClass( "ui-slider-" + this.orientation );
 				this._refreshValue();
-
+				this._refreshRange( value );
```

Comment:
Please restore this blank line, it's required by our style guide (all comments must be preceded by a blank line).